src/components/ScatterRow/styles.scss

```diff
@@ -28,6 +31,27 @@
     .d3-point {
       fill: $selected;
       cursor: pointer;
+
+      // return-to-normal animation
+      transition: all 0.5*$base-time ease-in-out;
+
+      &:hover {
+        // kick-off animation
+        transition: all 0.2*$base-time cubic-bezier(0.44, 0.23, 0.19, 0.98);
+      }
+
+      // growing animation
+      &.growing {
+        transition: all 0.8*$base-time cubic-bezier(0.44, 0.23, 0.19, 0.98);
+      }
+
+      transition-delay: 0.5*$base-time;
+
+      &.shrinking {
+        transition-delay: 0.75*$base-time;
+        transition-duration: 0.5*$base-time;
```

Comment:
It is easier to read when you give space between elements:
`0.75 * $base-time;`